requirements.txt

```diff
@@ -5,9 +5,9 @@ Flask-Assets==0.12
 Flask-DebugToolbar==0.7.1
 Flask-SeaSurf==0.1.13
 Flask==0.10
-gevent==1.0.2
+gevent==1.1.2
 gunicorn>=17.5,<=19.4.1
-hiredis==0.1.1
+hiredis==0.1.4
```

Comment:
That was only necessary on my side, not for the project. Changes in `requirements.txt` can be removed